PDBminer

```diff
@@ -726,49 +697,34 @@ def get_uniprot_sequence(uniprot_id, isoform):
     uniprot_numbering : A list of numerical values describing the residue position.
 
     """
-    print(f"FUNCTION: get_uniprot_sequence({uniprot_id}, {isoform})")
+    logging.debug(f"FUNCTION: get_uniprot_sequence({uniprot_id}, {isoform})")
     
     if isoform==1:
     
         try: 
             response = requests.post(f"https://rest.uniprot.org/uniprotkb/{uniprot_id}.fasta")
         except ConnectionError as e:
-            with open(f"results/{uniprot_id}/log.txt", "a") as textfile:
-                textfile.write(f"EXITING: Uniprot database API rejected for {uniprot_id}. \n")
-            exit(1) 
-        
-        if response.status_code == 200:
-            sequence_data=''.join(response.text)
-            Seq=StringIO(sequence_data)
-            pSeq=list(SeqIO.parse(Seq,'fasta'))
-            uniprot_sequence = str(pSeq[0].seq)
-            uniprot_numbering = list(range(1,len(uniprot_sequence)+1,1)) 
-
-        else:
-            with open(f"results/{uniprot_id}/log.txt", "a") as textfile:
-                textfile.write(f"EXITING: The canonical sequence could not be retrieved. \n")
+            logging.error(f"EXITING: Uniprot database API rejected for {uniprot_id}.")
```

Comment:
this would happen because we cannot connect to the API, not because the Api rejects our request (.e. the API doesn't return an error, we just cannot reach it), so I would change the error message